pkg/ingester/ingester.go

```diff
@@ -202,6 +202,8 @@ type Config struct {
 
 	ErrorSampleRate int64 `yaml:"error_sample_rate" json:"error_sample_rate" category:"advanced"`
 
+	// use-ingester-owned-series-for-limits was added in 2.12, but we keep it experimental until we decide, what is the correct behaviour
```

Comment:
nit to make it a godoc
```suggestion
	// UseIngesterOwnedSeriesForLimits was added in 2.12, but we keep it experimental until we decide, what is the correct behaviour
```